.github/workflows/end-to-end.yml

```diff
@@ -35,37 +35,81 @@ jobs:
       - name: checkout
         uses: actions/checkout@v2
       - name: install-bash
-        run: ./install.sh bash | tee log.txt && bash -lc 'cat $AM_PROMPT/.sha'
+        run: ./hack/install.sh bash
```

Comment:
@dmccaffery Why chose the name hack, I'm curiouse?

Reply:
don't actually understand the reasons for this, but its a growing trend in many of the OSS projects I participate in, mostly in the golang world. Probably a direct result of Kubernetes doing it:

https://github.com/kubernetes/kubernetes/tree/master/hack